src/quantum/target-status.jsx

```diff
@@ -6,6 +6,7 @@ import { statusLabels } from './constants';
 const TargetStatus = ({ notes }) => {
   const allStatus = new Map([['green', 0], ['yellow', 0], ['red', 0]]);
 
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
What's the default restricted syntax setting? Its something Armen might want to customize later, but it'd be good to see if there are alternatives to whats restricted in the meantime.

Reply:
I wasn't not sure about it. I will look into it in another PR.